readme.md

```diff
@@ -140,7 +177,12 @@ Instantiate an instructor named 'Andy' and call its `receivePresent` method with
 
 ```js
 // Your code starts here...
-
+var Instructor = function (name) {
+  this.name = name;
+  this.receivePresent = function(gift) {
+    console.log(this.name + " promptly drops the " + gift + " on the floor");
+  }
+}
```

Comment:
Close.  Need parens for the constructor function.  Where's the usage code?  Please review the answer key.